sale_report_salesperson_from_partner/views/sale_report_views.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="view_order_product_search" model="ir.ui.view">
+        <field name="inherit_id" ref="sale.view_order_product_search" />
+        <field name="model">sale.report</field>
+        <field name="arch" type="xml">
+            <filter name="User" position="before">
+                <filter
+                    string="Salesperson from Partner"
+                    name="user_from_partner_id"
+                    context="{'group_by':'user_from_partner_id'}"
+                />
+            </filter>
```

Comment:
You can add `<field name=‘user_from_partner_id’` to facilitate the search by this field.

Reply:
Added, thanks!